src/interpolate.jl

```diff
@@ -8,14 +8,14 @@ using OffsetArrays: OffsetVector
 ## ------ Akima Interpolation  ---------
 
 # struct used internally
-struct Akima{TF, T1<:AbstractVector{TF}, T2<:AbstractVector{TF}}
+struct Akima{TF1, TF2, T1<:AbstractVector{TF1}, T2<:AbstractVector{TF2}}
```

Comment:
This could be a breaking change if anyone references Akima as Akima{TF, T1, T2}.

Reply:
I'm fine with changing this though.

Reply:
yeah I guess technically breaking, but none of us are using it like that, and I don't see how to avoid it.